vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go

```diff
@@ -314,7 +314,7 @@ func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 		{
 			Message: "Advanced Commands:",
 			Commands: []*cobra.Command{
-				NewCmdApply(f, out, err),
+				NewCmdApply("kubectl", f, out, err),
```

Comment:
There doesn't seem to be one, in this file at least; closest thing would be to read the cobra command's `Use` string, which just seems to have "kubectl" [hardcoded in there](https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go#L238) as well.

Best thing would probably be to have [NewKubectlCommand](https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go#L235) receive a new argument `baseName string` and go from there.

Ultimate source of truth seems to [lie here](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cli.go#L289-L294) :)